ci/jenkins/test-vmc.sh

```diff
@@ -268,6 +268,23 @@ function copy_image {
   ${SSH_WITH_ANTREA_CI_KEY} -n capv@${IP} "sudo crictl images | grep '<none>' | awk '{print \$3}' | xargs -r crictl rmi"
 }
 
+function run_codecov {
+    flag=$1
+    file=$2
+    dir=$3
+    remote=$4
+    ip=$5
+
+    bash
+    unset GOVC_URL GOVC_USERNAME GOVC_PASSWORD
```

Comment:
I think instead of this, we should replace the 2 `bash` commands below with `env -i bash`, what do you think? This way even if we have some environment variables on the VM, they will be hidden. What do you think?

Reply:
Thank you. It is a good idea. Code updated.